pecos/xmc/xlinear/model.py

```diff
@@ -299,6 +299,53 @@ def split_model_at_depth(self, given_depth, reindex=False):
             )
         return split_model
 
+    @classmethod
+    def reconstruct_model(cls, parent_model, child_models, Y_ids_of_subtrees):
+        """Reconstruct the model with parent_model and child_models
+
+        Args:
+            parent_model: a XLinearModel that has model_chain of the
+                    top part of original model
+            child_models: A list of child models
+            Y_ids_of_subtrees: A list of index mapping to original labels from the child model output
```

Comment:
Please add the expected type of each argument.

Reply:
Please add explaination of `Y_ids_of_subtrees`. In particular, `Y_ids_of_subtrees[i,j]` is the final label index of local label `j` from model `i`.